### PR TITLE
Increases robot limb malfunction threshold

### PR DESCRIPTION
**Pull request by ACCount12**
Robot limbs now need 20 damage to start malfunctioning (up from 10), and the malfunction chances are lower.

### CODE REVIEW

File: code/modules/organs/external/_external.dm [hunk 1]
```diff
@@ -819,8 +819,9 @@ Note that amputating the affected organ does in fact remove the infection from t
 			return 1
 	return 0
 
+// Robotic limbs malfunction - handled by subtype
 /obj/item/organ/external/proc/is_malfunctioning()
-	return (BP_IS_ROBOTIC(src) && (brute_dam + burn_dam) >= 10 && prob(brute_dam + burn_dam))
```
Comment (MadmanMartian) [inline code/modules/organs/external/_external.dm]: Undo this change. BP_IS_ROBOTIC checks the organs nature, which can be changed without it being a subtype (Example being body modification, check body_modifications.dm, line 195 for example)

Comment (ACCount12) [reply code/modules/organs/external/_external.dm]: Now look at the line 192 in the same file.
```body_parts = list(BP_HEART, BP_LUNGS, BP_LIVER, BP_EYES)```
It's limited to some select INTERNAL organs. Actual loadout limbs replace the limb, complete with a /robotic path.
